src/gui/properties/peerlistwidget.cpp

```diff
@@ -72,7 +72,7 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     header()->setStretchLastSection(false);
     // List Model
     m_listModel = new QStandardItemModel(0, PeerListDelegate::COL_COUNT, this);
-    m_listModel->setHeaderData(PeerListDelegate::COUNTRY, Qt::Horizontal, tr("Country")); // Country flag column
+    m_listModel->setHeaderData(PeerListDelegate::COUNTRY, Qt::Horizontal, tr("Country / Region")); // Country flag column
```

Comment:
IMO, spaces are redundant here: Country/Region

Reply:
@Piccirello, are you going to fix it?

Reply:
Fixed.